package.json

```diff
@@ -17,7 +18,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged && yarn test"
```

Comment:
```yarn test``` :+1:

Reply:
yarn test took 16.951s on my machine. Having this on a pre-commit hook, and requiring that tests pass is going to be a productivity loss and will not scale well as more tests are added.

Github PR actions provide the same safety check.

Reply:
Please remove this change. It would need larger discussion. Github actions should cover test green-ness

Reply:
It just took 3-5 seconds on my machine to complete, however, I agree that it might take longer if the unit test cases increases. I believe it's more productivity loss to wait until Github Actions does the same job. Sometimes Github Actions or any other CICD tool might be down or running slow due to unavailable active agents or surge in processing request, it might delay outputs. Having it on pre-commit hooks on local setup is more productive during such cases. 

It's just an additional check and can be skipped using "--no-verify" if no functional changes made on PRs.

Reply:
@tstirrat - removed from pre-commit hooks for now.